FacadeApp/Program.cs

```diff
@@ -28,21 +35,21 @@ static async Task Main(string[] args)
                    dataDir = o.DataDir;
                });
 
-            Console.WriteLine("Path: " + golemPath);
-            Console.WriteLine("DataDir: " + (dataDir ?? ""));
+            logger.LogInformation("Path: " + golemPath);
+            logger.LogInformation("DataDir: " + (dataDir ?? ""));
 
-            await using (var golem = new Golem.Golem(golemPath, dataDir))
+            await using (var golem = new Golem.Golem(golemPath, dataDir, loggerFactory))
```

Comment:
I am not surre if it is correct approach in dotnet. I wanted to have logs distinguishable between various parts of code, so I pass `loggerFactory` which lets me to generate loggers using Class name prefix.